core/src/test/java/com/opensymphony/xwork2/DefaultUnknownHandlerManagerTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.opensymphony.xwork2;
+
+import com.opensymphony.xwork2.config.entities.ActionConfig;
+import com.opensymphony.xwork2.config.providers.SomeUnknownHandler;
+
+import java.util.ArrayList;
+
+/**
+ * Partial test to the DefaultUnknownHandlerManager to understand the relationship between Manager and Handlers.
+ *
+ */
+public class DefaultUnknownHandlerManagerTest extends XWorkTestCase {
```

Comment:
I tested `extends TestCase` and works. It seems you can downgrade to `TestCase` to save performance.

Reply:
I was following other test cases here. I will change it and push it if tests pass.